.gitignore

```diff
@@ -4,6 +4,7 @@ node_modules
 *.log
 npm-debug.log*
 dist/
+cypress/
```

Comment:
Litt voldsomt å ignorere hele cypress-mappa kanskje, men cypress/screenshots kan gjerne legges til 👍

Reply:
Og om man ønsker å legge til noe i gitignore som allerede er tracket i git, så må man utracke det som allerede er tracket.
Eksempel på hvordan det kan gjøre for en mappe/fil:
https://sufiyanyasa.com/blog/git-untrack-file/

Reply:
Gikk litt kjapt i svingene her. Det var cypress/screenshots som jeg mente å legge til :D